src/Selenium2Library/keywords/element.py

```diff
@@ -728,6 +748,16 @@ def _get_text(self, locator):
             return element.text
         return None
 
+    def _check_element_css_value(self, locator, prop, expected):
```

Comment:
Because this method is only used in the `Element Css Attribute Should Be` it might be better idea to move the code from here to the keyword.